openmc/summary.py

```diff
@@ -567,8 +543,21 @@ def make_opencg_geometry(self):
         if self.opencg_geometry is None:
             self.opencg_geometry = get_opencg_geometry(self.openmc_geometry)
 
-
     def get_nuclide_by_zaid(self, zaid):
+        """Return a Nuclide object given the 'zaid' identifier for the nuclide.
+
+        Parameters
+        ----------
+        zaid : int
+            1000*Z + A, where Z is the atomic number of the nuclide and A is the
+            mass number. For example, the zaid for U-235 is 92235.
+
+        Returns
+        -------
+        nuclide : openmc.nuclide.Nuclide
+            Nuclide matching the specified zaid
```

Comment:
Should any mention be made in routines like this one that ``None`` will be returned if no matching object is found?